tokenserver/util.py

```diff
@@ -22,7 +22,10 @@ def monkey_patch_gevent():
         from gevent import monkey
     except ImportError:
         return
-    monkey.patch_all()
+    try:
+        monkey.patch_all()
+    except AttributeError:
```

Comment:
This was failing with an internal error because the monkey patch was failing to find an attribute for the `gevent` object. 

This point is now moot since we're not using gevent, so I can pull this. Thanks!

Reply:
And, I spoke too soon: 
The error I get when running tests is:
```AttributeError: 'thread.RLock' object has no attribute '_RLock__owner'```

Reply:
Gotcha. From https://github.com/gevent/gevent/issues/1380 it sounds like this may be fixed in the latest alpha releases of gevent-1.5, but if we're going to avoid gevent anyway, then probably doesn't matter.

FWIW, I think I'd rather have this function fail if it fails to monkey-patch gevent correctly, rather than silently failing to do what it was asked. Perhaps we can instead deal with this by erroring out with a clean error message if someone tries to call this when running under pypy?

Reply:
True, but then none of the tests will work, because it absolutely will fail right now. I'm fine adding a warning or something, but otherwise we're dead in the water until gevent 1.5 lands.

Reply:
Can we e.g. default `tokenserver.monkey_patch_gevent` to False and/or configure it explicitly to False in the tests, or alternately, rip out all the gevent stuff and declare it no longer supported?

Reply:
`tokenserver/__init__.py` calls `monkey_patch_gevent()` if the runner is `nosetests` [without looking at any config](https://github.com/mozilla-services/tokenserver/blob/master/tokenserver/__init__.py#L13), so no, we can't.

Reply:
> tokenserver/__init__.py calls monkey_patch_gevent() if the runner is nosetests

Ugh. I wish my past self had left some comments about why he thought that was necessary. Spelunking the git history reveals a link to this old issue, which suggests that it's to work around some test nonsense with zmq that may not relevant anymore:

  https://github.com/mozilla-services/tokenserver/issues/12

But, just removing it without being sure that we know why it was added seems likely to anger Murphy.

I don't like leaving the program in a potentially half-monkeypatched-half-not state, but I think I'm OK with leaving the `except AttributeError` in here in order to get this out the door, perhaps with a comment to link to https://github.com/gevent/gevent/issues/1380 for reference for our future selves.

As an aside, https://github.com/mozilla-services/tokenserver/issues/12 makes me wonder if the half-monkeypatched-half-not state may be what's causing the tests to fail in CircleCI.